client/src/server.jsx

```diff
@@ -0,0 +1,123 @@
+import { createServer, Model, hasMany, belongsTo } from "miragejs"
+
+export function makeServer({ environment = "test" } = {}) {
```

Comment:
See: 
https://miragejs.com/docs/main-concepts/factories/#in-testing:~:text=In%20test%2C%20Mirage%20will%20load%20all%20your%20server%20config%2C%20but%20it%20will%20ignore%20your%20seeds.%20(It%20also%20sets%20the%20timing%20to%200%20for%20route%20handlers%20and%20hides%20logs%20from%20the%20console.)

```suggestion
export function makeServer() {
```